### PR TITLE
feat: GSX Integration

### PR DESCRIPTION
**Pull request by Lucky38i**
## Summary of Changes

Integrates GSX features with the aircraft custom systems including refueling, boarding and cargo. De-Ice will come at a later stage when we simulate the usefull of de-ice on aircraft performance. 

GSX Profile disables the rear cargo door and rear passenger door, and disables the option of the fuel menu opening. Doors were disabled as they can't currently be opened in-game. If you still want passengers/cargo to use these doors then you can re-enable them in the `Airplane Settings` menu on GSX.

Documentation will follow suit.

- [x] Fuel Synced
- [x] GSX Profile
- [x] Boarding Synced
- [x] Baggage/Freight Loading Synced
- [x] Deboarding Synced
- [x] Baggage/Freight Emptying Synced
- [x] Disable A32NX's chocks and cones and use GSX when sync enabled
- [x] Use A32NX's cabin noises outside of boarding and deboarding
- [ ] Migrate to Rust
## References

- GSX Manual on Development Interactions

## Additional context

Discord username (if different from GitHub): Lucky38i

## Testing instructions

There's a few things to be aware of when using GSX Sync. **PLEASE READ** each of these carefully, documentation is planned for release.

* There's a new page in the EFB settings labelled `3rd Party Options`. Inside there are two options to give you the freedom on what to sync with GSX. This activates the synchronisation. Without it GSX won't sync as expected. It's important to note various items and fields get disabled in the payload and fuel page with these options enabled.
* De-fueling Is NOT supported, GSX only checks if the fuel has changed to a number higher than the current value. If you call the refueling truck and attempt to defuel, the aircraft will defuel but GSX will nag you to change fuel and not actually do anything, constantly nagging you.
* Fueling is managed by A32NX, GSX simply acts as a visual indicator here which allows the user to still use Instant/Fast/Real options in the EFB. However, Fuel cannot be started until the fueling truck has connected the hose as shown by a grey start button. The fueling truck will remain and count for the duration of the fueling period set by A32NX, you will of course get a bill as usual.
* You must set your amount of passengers before GSX boarding, whether through simbrief sync in the EFB or manually in the payload page. Same goes for cargo. Once GSX boarding has been requested, these fields _CANNOT_ be edited until deboarding (meaning an entire flight)
* Boarding is managed by GSX (in a sense), this means you cannot adjust the boarding times in the EFB, you can only adjust boarding time, through GSX's own settings. If one physical passengers boards the aircraft through GSX, then A32NX's Boarding backend is made aware and acts accordingly. Passenger seats are still loaded in the same way they would without GSX Sync. 
* The same is regarded with Cargo/Baggage, however due to limitations with the simvars available from GSX, all stations are loaded at the same time. So if GSX says baggage is 33% loaded, all stations will be loaded to 33% of their given weight target. This removed a bit of the randomness to loading but the limitations is out of my hands.

An aircraft profile is provided, you don't need to do anything GSX picks up on this
* With this profile, the aft passenger door is disabled and the the aft cargo door as-well. This is because these doors can't be opened from the EFB. The aft service door is not disabled, but must be manually opened through the EFB. I'll try and notify the GSX team on the simvars to open this door. So you won't have to in the future.

- [ ] Ensure simbrief loading still performs as before in EFB
- [ ] Fueling
  - [ ] Test GSX properly plays the re-fueling animation when you begin refueling with the EFB
  - [ ] Test the refueling animation correctly stops when the EFB stops refueling the aircraft
  - [ ] Test the refueling animation is instant when you're using Instant refueling time
  - [ ] Ensure you can still refuel in-flight with GSX Fuel Sync enabled
- [ ] Payload
  - [ ] Set in the EFB how much passengers you want as-well as cargo, to notify GSX of these details. 
  - [ ] Ensure passenger current in EFB, matches GSX tooltip
  - [ ] Confirm as GSX baggage tooltip percentage increase, current cargo weight increase in EFB
  - [ ] Ensure baggage weight decreases when deboarding
  - [ ] Ensure passenger amount decreases when deboarding
  - [ ] Ensure passenger/cargo amount in EFB cannot be modified after boarding is requested and only appears again after deboarding is completed 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: src/instruments/src/EFB/Settings/Settings.tsx [hunk 1]
```diff
@@ -45,6 +46,7 @@ export const Settings = () => {
         { alias: t('Settings.AircraftOptionsPinPrograms.Title'), name: 'Aircraft Options / Pin Programs', component: <AircraftOptionsPinProgramsPage /> },
         { alias: t('Settings.SimOptions.Title'), name: 'Sim Options', component: <SimOptionsPage /> },
         { alias: t('Settings.Realism.Title'), name: 'Realism', component: <RealismPage /> },
+        { alias: t('Settings.GsxOptions.Title'), name: 'GSX Options', component: <GsxOptionsPage /> },
```
Comment (frankkopp) [inline src/instruments/src/EFB/Settings/Settings.tsx:49]: I'm not really thrilled about adding a whole page just for one 3rd party add-on. 
Is it possible to at least hide it when GSX is not installed (there is probably some LVAR indicating that it is installed)
Also maybe a new page "3rd Party Settings" (or something better)  would make more sense than a GSX only page.

**Review by frankkopp — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Settings/Settings.tsx (hunk 1 above)
Comment (Lucky38i) [reply src/instruments/src/EFB/Settings/Settings.tsx:49]: Sounds good to me, I'll do that

**Comment by 2hwk** [comment]
Note: ``A32NX_Boarding.js`` will have conflicts with #7442

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg [hunk 2]
```diff
@@ -0,0 +1,63 @@
+[aircraft]
+icaotype = A20N
+parkingbrakestest = (L:A32NX_PARK_BRAKE_LEVER_POS,bool) (L:A32NX_MODEL_WHEELCHOKS_ENABLED,bool) ||
```
Comment (Revyn112) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg:3]: Typo error: `A32NX_MODEL_WHEELCHOKS_ENABLED`
Regarding the model behaviour XML it should be `A32NX_MODEL_WHEELCHOCKS_ENABLED`

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js [hunk 3]
```diff
@@ -86,139 +91,241 @@ class A32NX_Boarding {
...
             }
-        }
 
-        // Sound Controllers
-        if ((currentPax < paxTarget) && boardingStartedByUser == true) {
-            await SimVar.SetSimVarValue("L:A32NX_SOUND_PAX_BOARDING", "Bool", true);
-            this.isBoarding = true;
+            this.loadPaxPayload();
+            this.loadCargoPayload();
+
```
Comment (2hwk) [inline flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js:197]: Tbh, while I was working on rebasing my PR, I'm not really sure I am keen on this design where we are essentially duplicating (with a if else) the logic driving boarding stations.

There should be a way to abstract this such that there wouldn't need to be repeated/duplicate code.

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js (hunk 3 above)
Comment (Lucky38i) [reply flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js:197]: The problem is the boarding logic isn't really duplicated, each for loop different between GSX and native. Then there's the problem of when those for loops are called. For native it's at the bottom after checking various things.

I can attempt to abstract this to a method that handles GSX vs native loading to reduce duplication, but the original if else will most likely remain

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg (hunk 2 above)
Comment (LeDerpLegend) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg:3]: Quality Assurance Trainee Report

Discord : LeDerpLegend#0109
Object of testing: #7695 
Tier of Testing : 1
Date : 16/01/2023

Testing Process:
Ensure simbrief loading still performs as before in EFB - GOOD

**Fueling**

- Test GSX properly plays the re-fueling animation when you begin refueling with the EFB - **OK**
- Test the refueling animation correctly stops when the EFB stops refueling the aircraft - **OK**
- Test the refueling animation is instant when you're using Instant refueling time - **OK**
- Ensure you can still refuel in-flight with GSX Fuel Sync enabled - **OK**

_Notes: Fuel button was NOT greyed out when requesting for a fuel truck, could fuel before truck arrived or hooked up._

**Payload**

- Set in the EFB how much passengers you want as-well as cargo, to notify GSX of these details. - **OK**
- Ensure passenger current in EFB, matches GSX tooltip - **OK**
- Confirm as GSX baggage tooltip percentage increase, current cargo weight increase in EFB - **FAIL**
- Ensure baggage weight decreases when deboarding - **FAIL**
- Ensure passenger amount decreases when deboarding - **FAIL**
- Ensure passenger/cargo amount in EFB cannot be modified after boarding is requested and only appears again after deboarding is completed - **FAIL**

_Notes: GSX PAX loaded super quickly in about 10 seconds, and I still have the ability to modify PAX and Payload between requesting boarding and fully deboarded._

Negatives:
- PAX and Payload connection with GSX not working as intended. 
- The ability to still modify fuel and payloads while GSX is preparing or between two stages should not be available. 
- Two catering vehicles still arrive at the plane, despite the front door not being able to open, however, the service still functions as normal.

Positives: 
- Only one baggage loader and cart spawned as intended, considering only one cargo bay door can be opened. (Restarting the Couatl service breaks this, and spawns two carts.)

Testing Result: NOT PASSED

Conclusions: Some GSX intergrations work as intended, but it seems there is not connection at all between the PAX/Payload EFB and GSX. GSX also seems to load PAX super fast. The blocking of fuel, payload, and passenger loading is not working as intended. I can still modify the fuel, passengers, and payload whenever, despite GSX's current position of vehicles and status of loading/unloading.

**Comment by LeDerpLegend** [comment]
Quality Assurance Trainee Report

Discord : LeDerpLegend#0109
Object of testing: #7695 
Tier of Testing : 1
Date : 16/01/2023

Testing Process:
Ensure simbrief loading still performs as before in EFB - GOOD

**Fueling**

- Test GSX properly plays the re-fueling animation when you begin refueling with the EFB - **OK**
- Test the refueling animation correctly stops when the EFB stops refueling the aircraft - **OK**
- Test the refueling animation is instant when you're using Instant refueling time - **OK**
- Ensure you can still refuel in-flight with GSX Fuel Sync enabled - **OK**

_Notes: Fuel button was NOT greyed out when requesting for a fuel truck, could fuel before truck arrived or hooked up._

**Payload**

- Set in the EFB how much passengers you want as-well as cargo, to notify GSX of these details. - **OK**
- Ensure passenger current in EFB, matches GSX tooltip - **OK**
- Confirm as GSX baggage tooltip percentage increase, current cargo weight increase in EFB - **FAIL**
- Ensure baggage weight decreases when deboarding - **FAIL**
- Ensure passenger amount decreases when deboarding - **FAIL**
- Ensure passenger/cargo amount in EFB cannot be modified after boarding is requested and only appears again after deboarding is completed - **FAIL**

_Notes: GSX PAX loaded super quickly in about 10 seconds, and I still have the ability to modify PAX and Payload between requesting boarding and fully deboarded._

Negatives:
- PAX and Payload connection with GSX not working as intended. 
- The ability to still modify fuel and payloads while GSX is preparing or between two stages should not be available. 
- Two catering vehicles still arrive at the plane, despite the front door not being able to open, however, the service still functions as normal.

Positives: 
- Only one baggage loader and cart spawned as intended, considering only one cargo bay door can be opened. (Restarting the Couatl service breaks this, and spawns two carts.)

Testing Result: NOT PASSED

Conclusions: Some GSX intergrations work as intended, but it seems there is not connection at all between the PAX/Payload EFB and GSX. GSX also seems to load PAX super fast. The blocking of fuel, payload, and passenger loading is not working as intended. I can still modify the fuel, passengers, and payload whenever, despite GSX's current position of vehicles and status of loading/unloading.

**Review by frankkopp — APPROVED** [review]

**Comment by frankkopp** [comment]
I have uploaded your flyPad translation keys to Localazy:
```json
 "ThirdPartyOptions": {
      "Title": "3rd Party Options",
      "GsxFuelEnabled": "GSX Fueling Synchronization",
      "GsxPayloadEnabled": "GSX Payload Synchronization"
    },
```

File: src/instruments/src/EFB/Settings/Settings.tsx (hunk 1 above)
Comment (Lucky38i) [reply src/instruments/src/EFB/Settings/Settings.tsx]: resolved [c764914](https://github.com/flybywiresim/a32nx/pull/7695/commits/c764914ebbb62b43414416c946df0837204de948)

**Comment by Lucky38i** [comment]
> Quality Assurance Trainee Report
> 
> Discord : LeDerpLegend#0109 Object of testing: #7695 Tier of Testing : 1 Date : 16/01/2023

Thanks for the test :) @LeDerpLegend 

> Negatives:
> 
>     * Just visually, two luggage cars and two catering trucks will arrive, but only one door will open, I get that's probably a model thing nothing we can do, functionality still works. Nothing to worry about.

The aft service door does indeed work so that should work. You might need to manually open it in the EFB. Perhaps GSX is unaware of the simvar to do it on it's own. The aft cargo door cannot be opened and so that is disabled in accordance with the provided GSX profile So only one luggage car should arrive. Make sure you don't already have a custom profile.

>     * If de-fueling is required, GSX as stated will not do anything, instead of nagging, the truck seems to connect, then after a short time, just disconnects and leaves.
> 
This is interesting, couldn't get it to stop nagging on my end.
> 
> Conclusion: Well done on this, honestly, it's something I've been looking forward to for a while. I think this could be cleaned up a bit in terms of UI and some instructions (like making sure to set Payload ahead of requesting boarding, or buttons becoming disabled.) Overall, good job.

I'll be making documentation as-well.

File: flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js (hunk 3 above)
Comment (Lucky38i) [reply flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Boarding.js]: resolved [3c420ef](https://github.com/flybywiresim/a32nx/pull/7695/commits/3c420ef6b7e05dbdafcd7d1d457722de51d7a24b)

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg (hunk 2 above)
Comment (Lucky38i) [reply flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg]: resolved [3c420ef](https://github.com/flybywiresim/a32nx/pull/7695/commits/3c420ef6b7e05dbdafcd7d1d457722de51d7a24b)

**Comment by LeDerpLegend** [comment]
> > Quality Assurance Trainee Report
> > Discord : LeDerpLegend#0109 Object of testing: #7695 Tier of Testing : 1 Date : 16/01/2023
> 
> Thanks for the test :) @LeDerpLegend
> 
> > Negatives:
> > ```
> > * Just visually, two luggage cars and two catering trucks will arrive, but only one door will open, I get that's probably a model thing nothing we can do, functionality still works. Nothing to worry about.
> > ```
> 
> The aft service door does indeed work so that should work. You might need to manually open it in the EFB. Perhaps GSX is unaware of the simvar to do it on it's own. The aft cargo door cannot be opened and so that is disabled in accordance with the provided GSX profile So only one luggage car should arrive. Make sure you don't already have a custom profile.
> 
This has to do with the front service door. The aft service door opens up as intended automatically, however the front one, visually at least, remains closed, but the catering truck still operates normally. Just some dude who has the power to phase through walls I guess. 

I know that the aft cargo door can not be opened, I should state that this issue seems to happen when arriving back at an airport, or restarting GSX's connection service. I can never remember how to spell it but something along the lines of Colgut or something.
**I have an issue of the GSX service quitting after taking off.** I have no idea if this is an issue with GSX itself or something on my PC, but it always seems to quit on itself, requiring me to restart it. For me, spawning in at an airport, will correctly yield one loader and cart. 

I have no custom profiles for the A320 or KLAS
> > ``` 
> > * If de-fueling is required, GSX as stated will not do anything, instead of nagging, the truck seems to connect, then after a short time, just disconnects and leaves.
> > ```
> 
> This is interesting, couldn't get it to stop nagging on my end.
> 
It may have to do with some settings with GSX fueling. I set it to fuel at least for 5 seconds before disconnecting. I wonder if this changes anything. If you want I can send an image of my settings later. I have had no issues with it nagging me when attempting to defuel, it just plays the animation, fuels for a little bit (with audio, it doesn't change the fuel load in the plane whatsoever.) Then they pack up and leave like normal. 
> > Conclusion: Well done on this, honestly, it's something I've been looking forward to for a while. I think this could be cleaned up a bit in terms of UI and some instructions (like making sure to set Payload ahead of requesting boarding, or buttons becoming disabled.) Overall, good job.
> 
> I'll be making documentation as-well.

Very well.

**Comment by ldee247** [comment]
Hi, I'm requesting authorization to download files I'm a Headwind a339 pilot. Saw the link through the discord channel for Headwind, thanks in advance.

File: .github/CHANGELOG.md [hunk 4]
```diff
@@ -26,108 +26,109 @@
...
+89. [ENGINE] Fix Engine Flex Calculations - @Taz5150 (TazX [Z+2]#0405)
+90. [FLIGHTMODEL] Flight Model update - @donstim (donbikes#4084)
+91. [HYD] Fix incorrect gain in down pitch commands - @Crocket63 (crocket)
+92. [PFD] Improve appearance of L/DEV and V/DEV scales - @tracernz (Mike)
+93. [HYD] Implemented Electro Hydrostatic actuators - @Crocket63 (crocket)
+94. [MISC] Change 2 way overhead switches to be also controllable with mouse wheel up/down @patsta32 (Pat)
+95. [FAC] Move Speedscale computation to FAC - @lukecologne (luke)
+96. [EFB] Added deboarding button to flyPad Payload - @frankkopp (Frank Kopp)
+97. [EFB] Improved simbridge-client connection handling - @frankkopp (Frank Kopp)
+98. [HYD] Tilting gear mechanism - @Crocket63 (crocket)
```
Comment (2hwk) [inline .github/CHANGELOG.md:131]: Please do not change this, 1. works for bulleting. 
```suggestion
2. [MCDU] Allow Wind Request from Simbrief flight plan - @USA-RedDragon (Jacob McSwain)
3. [HYD] Fix fluid return handling of actuators - @Crocket63 (crocket)
4. [ENGINE] Fuel persistency between each flight - @juliansebline (Julian Sebline#8476)
5. [HYD] Now allowing reverting gravity gear extension - @Crocket63 (crocket)
6. [TCAS] Fixed issue when turning to STBY while RA is issued - @2hwk (2Cas#1022)
7. [GPWS] Alt callouts now finish before altitude is reached - @2hwk (2Cas#1022)
8. [ENGINE] Improved engine model at TOGA - @Taz5150 (TazX [Z+2]#0405)
9. [FLIGHTMODEL] Improve pitch trim and manual braking level - @donstim (donbikes#4084)
10. [ATSU] Fix cargo weight is not set on manually PAX - @Revyn112 (Revyn112#1010)
11. [CDSS] When off no longer displays door video - @davidwalschots (David Walschots)
12. [FMGC] Map leg course with station declination where required - @tracernz (Mike)
13. [HYD] Gear system now supports spawning in flight gear up - @Crocket63 (crocket)
14. [MCDU] Added imperial runway length - @derl30n (Leon)
15. [MCDU] Refactor and improve input handling and conversions in MCDU W&B - @sidnov (Sid)
16. [MCDU] Improved arrivals/departure page scrolling, more true-to-life behaviour and cosmetic apperance - @2hwk ( 2Cas#1022)
17. [PFD] Show yellow GS reference line in correct conditions - @saschl (saschl#9432)
18. [ECAM] Fix erroneous SLATS NOT IN T.O CONFIG warning during flaps 3 takeoff - @beheh (Benedict Etzel)
19. [MODEL] Improved rivet mesh with more variation and detail - @Grinde (Grinde#4017)
20. [ELEC] Make battery voltmeters update one digit at a time - @beheh (Benedict Etzel)
21. [EWD] E/WD visual improvements - @lukecologne (luke)
22. [SFCC] Add SFCC bus outputs - @lukecologne (luke)
23. [EWD] Use FPPU angles for flaps/slats display - @lukecologne (luke)
24. [FMGC] Basic RNP at or below 0.3 support - @tracernz (Mike)
25. [ND] Add TERR ON ND peaks mode map visualization - @Sven [de en] (Sven Czarnian)
26. [SD] Improve F/CTL and WHEEL SD pages visuals - @lukecologne (luke)
27. [MCDU] Added formatter to improve text alignment and ease integration - @derl30n (Leon)
28. [EFB] Fix default unit to match UI and other consumers - @tracernz (Mike)
29. [HYD] Added possibility to lock or limit hydraulic actuators velocity - @Crocket63 (crocket)
30. [HYD] Added a new THS mechanism simulation - @Crocket63 (crocket)
31. [HYD] More prox sensors and gear actuator failures - @Crocket63 (crocket)
32. [MCDU] Added 4:3 aspect ratio compatibility to remote mcdu client - @tyler58546 (tyler58546)
33. [HYD] Fixed too slow leak measurement valves operation - @Crocket63 (crocket)
34. [ISIS] Added temporary ISIS font with arrows - @aweissoertel (Alexibexi#7550)
35. [PFD] Improve PFD barberpole rendering and behaviour - @lukecologne (luke)
36. [HYD] Fixed actuator position control demand consistency - @Crocket63 (crocket)
37. [FMGC] Only emit decel point when an approach is selected - @tracernz (Mike)
38. [HYD] Fixed SFCC computer failing to send FPPU commands - @Crocket63 (crocket)
39. [MCDU] Fix padding of arc radii on F-PLN - @tracernz (Mike)
40. [FMGC] Allow stringing of STARs with non-runway approaches - @tracernz (Mike)
41. [SD] Improve COND page visuals, fix some visual bugs on the SD - @lukecologne (luke)
42. [HYD] Aerodynamic forces applied on gear system - @Crocket63 (crocket)
43. [HYD] Fixed Lgciu state machine when reverting gravity extension - @Crocket63 (crocket)
44. [FMGC] Fix unnatural turn direction for path capture transition - @tracernz (Mike)
45. [FMGC] Fix unnatural turn direction for course capture transition - @tracernz (Mike)
46. [FMGC] Update spoiler CD for MSFS SDK clarification - @donstim (donbikes#4084)
47. [HYD] Actuators damping chambers can be asymetrical - @Crocket63 (crocket)
48. [FMGC] Implement procedure turns (PI leg) - @tracernz (Mike)
49. [ECAM] Move EWD to correct AC bus - @tracernz (Mike)
50. [FMGC] Fix inbound leg time for holds - @tracernz (Mike)
51. [MCDU] Improved visuals of Init-A and Init-B page - @derl30n (Leon)
52. [MODEL] Added new animated gear gravity extension handle- @tyler58546 (tyler58546), @MoreRightRudder (Mike), @Crocket63 (crocket), @Lantarius
53. [HYD] Custom Lvar for gear lever to fix ground collision bug - @Crocket63 (crocket)
54. [HYD] Randomised per actuator flow restrictions at plane init - @Crocket63 (crocket)
55. [MCDU] Hide stored elements on A/C Status when there are none - @tracernz (Mike)
56. [FMGC] Fix ident for CD legs - @tracernz (Mike)
57. [HYD] Refactor plane accelerations and added filtering to handle ground collisions - @Crocket63 (crocket)
58. [FMGC] Improve procedure loading performance - @tracernz (Mike)
59. [EFB] Added Payload Loading to EFB - @2hwk (2Cas#1022)
60. [MODEL] Remove blue highlights in lock mode - @tracernz (Mike)
61. [RMP] Fixed the behavior of the RMPs and ACPs (VHFs only) - Julian Sebline (Julian Sebline#8476 on Discord)
62. [PFD] Added option to sync FD/LS buttons between CPT and FO - @Eagle941 (Joe)
63. [EFB] Add automatic loading of a lighting preset - @SpiritZephyr (Ben)
64. [COND] Connect Air Conditioning System to Electrical System - @mjuhe (Miquel Juhe)
65. [HYD] Default discrete of failed ADIRs impacts gear safety valve correctly - @Crocket63 (crocket)
66. [ADDON] Simbridge Integration - @lucky38i (Alex)
67. [SD] Visual and functional improvements/fixes to the HYD SD page - @lukecologne (luke)
68. [HYD] Added optional auxiliary hydraulic section in core hydraulic circuits - @Crocket63
69. [EFB] Added SimBridge Health Check icon to Status Bar - @frankkopp (Frank Kopp)
70. [HYD] Fixed opened gear door wobbling - @Crocket63
71. [HYD] Fix gear extending whith dual lgciu power loss - @Crocket63 (crocket)
72. [EFB] flyPad pushback option to ignore controller inputs - @frankkopp (Frank Kopp)
73. [FCTL] Add realistically simulated F/CTL computers: ELAC/SEC/FCDC/FAC - @lukecologne (luke)
74. [HYD] Accelerations impacts hydraulic reservoir fluid movement - @Crocket63 (crocket)
75. [FLIGHTMODEL] Add SU10 crosswind parameters - @donstim (donbikes#4084)
76. [MODEL] Visible pilot avatars via flyPad setting - @frankkopp (Frank Kopp)
77. [MODEL] Fixed screens flickering in SU10 - @tyler58546 (tyler58546)
78. [EFB] Added Local Files support - @ErickSharp (Erick Torres) @frankkopp (Frank Kopp)
79. [BLEED] Added custom Wing Anti-Ice model - @omrygin, @Eagle941 (Joe)
80. [MISC] Fix FDR data for support analysis - @frankkopp (Frank Kopp)
81. [LIGHTS] Increased max brightness of cockpit displays - @frankkopp (Frank Kopp)
82. [EFB] Improved Aircraft Presets procedures - @frankkopp (Frank Kopp)
83. [FMGC] Make DES SPD LIM available prior to descent - @tracernz (Mike)
84. [LIGHTS] Redone cockpit emissives panels and ambient lighting -@FinalLightNL (FinalLight)
85. [FMGC] Rework f-pln departure page - @tracernz (Mike)
86. [MCDU] Support coroute loading and viewing - @lucky38i (Alex)
87. [HYD] Fixed autobrake controller tuning - @Crocket63 (crocket)
88. [ENGINE] Improved engine primary parameters performance (EGT & Fuel Flow) - @Taz5150 (TazX [Z+2]#0405)
89. [ENGINE] Fix Engine Flex Calculations - @Taz5150 (TazX [Z+2]#0405)
90. [FLIGHTMODEL] Flight Model update - @donstim (donbikes#4084)
91. [HYD] Fix incorrect gain in down pitch commands - @Crocket63 (crocket)
92. [PFD] Improve appearance of L/DEV and V/DEV scales - @tracernz (Mike)
93. [HYD] Implemented Electro Hydrostatic actuators - @Crocket63 (crocket)
94. [MISC] Change 2 way overhead switches to be also controllable with mouse wheel up/down @patsta32 (Pat)
95. [FAC] Move Speedscale computation to FAC - @lukecologne (luke)
96. [EFB] Added deboarding button to flyPad Payload - @frankkopp (Frank Kopp)
97. [EFB] Improved simbridge-client connection handling - @frankkopp (Frank Kopp)
98. [HYD] Tilting gear mechanism - @Crocket63 (crocket)
```
Please undo this change, 1. works for the changelog bulleting.

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Comment by Lucky38i** [comment]
> Hi, I'm requesting authorization to download files I'm a Headwind a339 pilot. Saw the link through the discord channel for Headwind, thanks in advance.

I have no idea what you're asking for here. This is a PR for the A32NX, I no idea this'll work with Headwind, speak to the people there.

**Review by 2hwk — APPROVED** [review]

File: flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+icaotype = A20N
+parkingbrakestest = (L:A32NX_PARK_BRAKE_LEVER_POS,number)
+nosegear = 8.26
+refueling = 0
+battery = 1
+pushbackraise = 1
+pushbackdummyevent = 0
+pushbackcheckengines = 0
+iscargo = 0
+trafficcones = 1
```
Comment (Saschl) [inline flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/GSX.cfg:11]: I guess we should disable GSX cones as we have our own. What do you think?

**Comment by choi-choi** [comment]
Tried with experimental version, enabled 3rd party GSX things. Simbrief used.

However the fuel truck always loads too slowly, even if i move the timings to x10. FBW is already done 5 mins ago, loading less than 3000kg of fuel. I always load real time.

Tried this with progressive fueling and detect 3rd party fueling checked.

For the passengers, the EFB of FBW has no buttons available, nothing also happens there and is basically dead. I assume they load..

**Comment by brewers10** [comment]
Quality Assurance Trainee Report

Discord          : brewers_10#5706
Object of testing: #7695 
Tier of Testing  : 2
Date             : 08/02/2023

Testing Process: 
 
Loaded into Stand 17B at KLAX for a flight to KSFO. Followed instructions to a tee: results below :)

Ensure simbrief loading still performs as before in EFB - **OK**
 
Fueling
 Test GSX properly plays the re-fueling animation when you begin refueling with the EFB - **OK**
 Test the refueling animation correctly stops when the EFB stops refueling the aircraft - **OK**
 Test the refueling animation is instant when you're using Instant refueling time - **OK**
 Ensure you can still refuel in-flight with GSX Fuel Sync enabled - **OK**

 Payload
 Set in the EFB how much passengers you want as-well as cargo, to notify GSX of these details. - **OK**
 Ensure passenger current in EFB, matches GSX tooltip - **OK** (tested with normal, dense and very dense)
 Confirm as GSX baggage tooltip percentage increase, current cargo weight increase in EFB - **OK**
 Ensure baggage weight decreases when deboarding - **OK**
 Ensure passenger amount decreases when deboarding - **OK**
 Ensure passenger/cargo amount in EFB cannot be modified after boarding is requested and only appears again after deboarding is completed

Negatives:       
Had to restart Couatl twice to get rid of the GSX bug where it decides to board around 10 passengers in one block. This messed up the planes passenger number when boarding was complete (only had 108 Pax compared to the planned 169). This is a GSX bug and nothing to do with the aircraft however.

Testing Results: 
Passed

Conclusions: 
Very very good work, this will certainly make people's lives easier when using GSX and the A32NX together. Top work to all involved!

Media:       
Microsoft Flight Simulator Screenshot 2023 02 08 - 18 53 03 49
Microsoft Flight Simulator Screenshot 2023 02 08 - 18 53 09 26
Microsoft Flight Simulator Screenshot 2023 02 08 - 18 53 12 38
Microsoft Flight Simulator Screenshot 2023 02 08 - 18 53 20 49

**Comment by Keight8** [comment]
would that be possible on deboarding to read the payload data from the current payload state instead of simbrief data? currently during the turnaround it ends up with messed payload.  for instance if you loaded 160 pax on the 1st leg,  while in cruise planned the 2nd leg where there are say 120 pax, then landed and requested deboarding it will disembark 120 pax instead of original 160 and 40 will remain.

**Pull request merged**